trace_test.go

```diff
@@ -125,7 +125,7 @@ type traceStats struct {
 }
 
 func (t *traceStats) process(evt *pb.TraceEvent) {
-	//fmt.Printf("process event %s\n", evt.GetType())
+	// fmt.Printf("process event %s\n", evt.GetType())
```

Comment:
could have removed this commented code

Reply:
tried to minimize the code impact on the rest of the code so it will help me doing the next merges
This was done automatically by the go fmt